docs/understanding-tracking-design/versioning-your-data-structures/cloud/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Versioning Data Structures in BDP Cloud"
+date: "2023-03-01"
+sidebar_position: 10
+---
+
+# Versioning in BDP Cloud
+
+If you are a BDP Cloud customer, you will have used our Data Structure Builder to create your custom Data Structure for your Events and Entities.
+
+![](../../managing-data-structures-with-data-structures-builder/images/data-structures-2.png)
+
+**Breaking and non-breaking changes**
+
+At the point of publishing a schema, there are two options for versioning your data structure.
+- **Non-breaking** - a non-breaking change is backward compatible with historical data and your loader.
```

Comment:
```suggestion
- **Non-breaking** - a non-breaking change is backward compatible with historical data and your warehouse loader.
```

Reply:
Or something that clarifies what we are talking about.

Reply:
👍